README.md

```diff
@@ -4,18 +4,628 @@
+      },
+      title: {
+        type: Sequelize.DataTypes.STRING(50),
+        allowNull: false,
+      },
+      done: {
+        type: Sequelize.DataTypes.BOOLEAN,
+        defaultValue: false,
+        allowNull: false,
+      },
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable("tasks");
+  }
+};
+```
+
+The above asks Sequelize to create a table `tasks`.
+
+To imagine what we just did look at a `tasks` table which can be created using above code.
+
+| guid           | title              | done |
+| ---------------| -------------------|------|
+| 235b6988-78..  | Watch NodeConf EU  | true |
+| a37e3990-41..  | Build Rome         | false|  
+
+
+3. Migrate
+
+```
+npx sequelize-cli db:migrate
+```
+
+The above will create a table called `tasks` in the configured database.
+
+### Model
+
+Since we have the tables in database now created, lets write the code which represents and connects to the corresponding table.
+
+Create a Model `Task.ts` in `src/models/` directory with the following code.
+
+`Task.ts`
+
+```
+import {
+    Table,
+    Model,
+    Column,
+    PrimaryKey,
+    BeforeCreate,
+  } from "sequelize-typescript";
+  import * as uuid from "uuid";
+  
+  @Table({ tableName: "tasks", timestamps: false, paranoid: false })
+  export class Task extends Model<Task> {
+    @PrimaryKey
+    @Column
+    guid: string;
+  
+    @Column
+    title: string;
+  
+    @Column
+    done: boolean;
+  
+    @BeforeCreate
+    static generateGuid(instance) {
+      if (!instance.guid) {
+        instance.guid = uuid.v4();
+      }
+    }
+  }
+  ```
+
+In the above Model, Column decorator defines the columns of table ie `guid`, `title`, `done`. The generateGuid static function makes sure to generate a guid for the task before creating a new one.
+
+### Testing
+
+Now that we have created our first model, lets start testing the functionality.
+
+In the `__tests__/models` directory create a file called `task.ts` and add the following code:
+
+```
+import { Process } from "actionhero";
+import { Task } from "../../src/models/Task";
+const actionhero = new Process();
+
+let api;
+
+describe("Model", () => {
+  describe("task", () => {
+    beforeAll(async () => {
+      api = await actionhero.start();
+    });
+
+    afterAll(async () => {
+      await actionhero.stop();
+    });
+
+    beforeAll(async () => {
+      Task.destroy({ truncate: true });
+    });
+
+    test("New Task gets created", async () => {
+      const task = await Task.create({
+        title: "Test Task",
+      });
+      expect(task).toHaveProperty("title");
+      expect(task).toHaveProperty("guid");
+      expect(task).toHaveProperty("done");
+      expect(task.title).toBe("Test Task");
+      expect(task.done).toBe(false);
+    });
+  });
+});
+
+```
+
+This code is self explanatory but for further clarity the above test checks if a new task gets created in the database with the required properties.
```

Comment:
Please try to avoid language like "self explanatory" - this could be insulting to a new reader who doesn't understand it (this is likely a cultural thing).  I tend to say something like:

> This code checks if a new task gets created in the database with the required properties.  To learn more about Jest tests, visit the [jest website](https://jestjs.io/)

Reply:
Got it. Fixed.